### PR TITLE
Fix: Update Ghost Bestiary Tiers

### PR DESCRIPTION
**Pull request by lunaynx**
## What
Updated Ghost Counter bestiary tiers to reflect Hypixel reducing the max kills needed from 250K to 100K.

Per-tier info taken from <https://wiki.hypixel.net/Bestiary#Tier_Scaling>.

## Changelog Fixes
+ Updated Ghost Counter bestiary tiers to reflect Hypixel reducing the max kills needed from 250K to 100K. - Luna

### CODE REVIEW

File: src/main/java/at/hannibal2/skyhanni/config/features/combat/ghostcounter/textformatting/BestiaryFormattingConfig.java [hunk 1]
```diff
@@ -30,7 +30,7 @@ public class BestiaryFormattingConfig {
     @ConfigOption(name = "Progress to Max", desc = "Text to show progress when the §eMaxed Bestiary §7option is §aON\n" +
         "§e%currentKill% §7is replaced with your current total kill.")
     @ConfigEditorText
-    public String showMax_progress = "%currentKill%/250k (%percentNumber%%)";
+    public String showMax_progress = "%currentKill%/100k (%percentNumber%%)";
```
Comment (CalMWolfs) [inline src/main/java/at/hannibal2/skyhanni/config/features/combat/ghostcounter/textformatting/BestiaryFormattingConfig.java:33]: THis needs a config fix

**Review by CalMWolfs — CHANGES_REQUESTED** [review]
Also this really should just source data from the neu repo for future (but prob out of the scope of this pr because i think empa is working on a rewrite of it but idk)

**Comment by ItsEmpa** [comment]
> Also this really should just source data from the neu repo for future (but prob out of the scope of this pr because i think empa is working on a rewrite of it but idk)

i mean the main thing missing from it is the config migration which i have no idea how to do

File: src/main/java/at/hannibal2/skyhanni/config/features/combat/ghostcounter/textformatting/BestiaryFormattingConfig.java (hunk 1 above)
Comment (hannibal002) [reply src/main/java/at/hannibal2/skyhanni/config/features/combat/ghostcounter/textformatting/BestiaryFormattingConfig.java:33]: done

**Pull request merged**